mmcls/datasets/pipelines/transforms.py

```diff
@@ -1120,12 +1121,20 @@ def __call__(self, results):
         # dict to albumentations format
         results = self.mapper(results, self.keymap_to_albu)
 
+        if 'gt_label' in results:
+            if isinstance(results['gt_label'], list):
+                results['gt_label'] = np.array(results['gt_label'])
+            results['gt_label'] = results['gt_label'].astype(np.int64)
+
+        # save gt_label incase Albu modify it.
```

Comment:
I think it may be better to mv this line before line1122. Since users may change the `gt_label` to another key name.


```
1. backup gt_label
2. Renames keys according to keymap provided.
3.  aug
4. revert key
5. recover backup gt_label
```

Reply:
Done